src/login/logind-user.c

```diff
@@ -891,7 +892,13 @@ int config_parse_user_tasks_max(
         assert(rvalue);
         assert(data);
 
-        /* First, try to parse as percentage */
+        /* Unlimited? */
+        if (isempty(rvalue) || streq(rvalue, "infinity")) {
```

Comment:
Hm, looks good in general. My only question is whether TasksMax= should reset to the default value or the infinity.